src/julia/options.py

```diff
@@ -168,8 +205,12 @@ def specified(self):
     def as_args(self):
         args = []
         for (desc, value) in self.specified():
-            args.append(desc.cli_argument_name())
-            args.append(value)
+            if value is None:
+                ...
```

Comment:
I added this new logic to avoid adding arguments if the value is unset. Not sure why this wasn't set up before - @mkitti or @marius311 any idea why?